### PR TITLE
Add a note about installing before testing

### PR DESCRIPTION
**Pull request by dmtucker**
See #914.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -48,6 +48,8 @@ Then you can run the testsuite with::
 
     py.test
 
+- Note: Werkzeug must also be installed for all tests to pass.
```
Comment (untitaker) [inline CONTRIBUTING.rst]: Better show how to install werkzeug (`pip install -e .`)

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -48,6 +48,10 @@ Then you can run the testsuite with::
 
     py.test
 
+- Note: Werkzeug must also be installed for all tests to pass::
```
Comment (untitaker) [inline CONTRIBUTING.rst]: Sorry for doing this review chunked.

I think this note should be after the first paragraph where virtualenv is linked, so the steps the dev has to perform are in order.

Comment (dmtucker) [inline CONTRIBUTING.rst]: Ah, good point... all good, that gives me a chance to fix my author/committer email mismatch anyways.

**Comment by untitaker** [comment]
Thanks!

**Pull request merged**